.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -256,6 +256,137 @@ def ACCHostVerificationTest(String version, String build_type) {
+    */
+    stage("ACC-1804 Generate Quote") {
+        node(AGENTS_LABELS["acc-ubuntu-18.04"]) {
+            timeout(GLOBAL_TIMEOUT_MINUTES) {
+                cleanWs()
+                checkout scm
+
+                println("Generating certificates and reports ...")
+                def task = """
+                           cmake ${WORKSPACE} -G Ninja -DCMAKE_BUILD_TYPE=${build_type} -Wdev
+                           ninja -v
+                           pushd tests/host_verify/host
+                           openssl ecparam -name prime256v1 -genkey -noout -out keyec.pem
+                           openssl ec -in keyec.pem -pubout -out publicec.pem
+                           openssl genrsa -out keyrsa.pem 2048
+                           openssl rsa -in keyrsa.pem -outform PEM -pubout -out publicrsa.pem
+                           ../../tools/oecert/host/oecert ../../tools/oecert/enc/oecert_enc --cert keyec.pem publicec.pem --out sgx_cert_ec.der
+                           ../../tools/oecert/host/oecert ../../tools/oecert/enc/oecert_enc --cert keyrsa.pem publicrsa.pem --out sgx_cert_rsa.der
+                           ../../tools/oecert/host/oecert ../../tools/oecert/enc/oecert_enc --report --out sgx_report.bin
+                           popd
+                           """
+                oe.ContainerRun("oetools-full-${version}:${DOCKER_TAG}", "clang-7", task, "--cap-add=SYS_PTRACE --device /dev/sgx:/dev/sgx")
+
+                def ec_cert_created = fileExists 'build/tests/host_verify/host/sgx_cert_ec.der'
+                def rsa_cert_created = fileExists 'build/tests/host_verify/host/sgx_cert_rsa.der'
+                def report_created = fileExists 'build/tests/host_verify/host/sgx_report.bin'
+                if (ec_cert_created) {
+                    println("EC cert file created successfully!")
+                } else {
+                    error("Failed to create EC cert file.")
+                }
+                if (rsa_cert_created) {
+                    println("RSA cert file created successfully!")
+                } else {
+                    error("Failed to create RSA cert file.")
+                }
+                if (report_created) {
+                    println("SGX report file created successfully!")
+                } else {
+                    error("Failed to create SGX report file.")
+                }
+
+                stash includes: 'build/tests/host_verify/host/*.der,build/tests/host_verify/host/*.bin', name: "linux_host_verify-${version}-${build_type}-${BUILD_NUMBER}"
+            }
+        }
+    }
+
+    /* Linux nonSGX stage. */
+    stage("Linux nonSGX Verify Quote") {
+        node(AGENTS_LABELS["ubuntu-nonsgx"]) {
+            timeout(GLOBAL_TIMEOUT_MINUTES) {
+                cleanWs()
+                checkout scm
+                unstash "linux_host_verify-${version}-${build_type}-${BUILD_NUMBER}"
+                def task = """
+                           cmake ${WORKSPACE} \
```

Comment:
Instead of building the SDK, is it possible to just stach the `host_verify` executable as well and then unstash it here and test it?

Reply:
We'd like to ensure `host_verify` can be built after the package installation, so that's why we do it after the package installation instead of stashing the executable and unstashing it on another machine.